core/bones/select.py

```diff
@@ -67,7 +66,9 @@ def singleValueSerialize(self, val, skel: 'SkeletonInstance', name: str, parentI
             return val.value
         return val
 
-    def singleValueFromClient(self, value, skel, name, origData):
+    def singleValueFromClient(self, value: Any, skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
```

Comment:
Is it necessary to provide full type annotations in sub-classes as well?
I would like to keep maintenance overhead low, if possible.